### PR TITLE
Feature/public scripts

### PR DESCRIPTION
**Pull request by ShannonKalei**
Just some last minute release stuff 😅 
- Couple typo fixes
- Added a new public scripts doc to be more fleshed out in the future 
- Linked the new public scripts doc in the release notes

### CODE REVIEW

**Review by gdamron — APPROVED** [review]
found a typo

File: docs/Scripting/PublicScripts.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+id: PublicScripts
+title: Public Scripts
+---
+
+
+Enklu Cloud comes with an ever-expanding library of public scripts. These scripts are available to all Enklu Cloud accounts and may be used in any experience.
+
+## Script Library
+
+The script library is located in the panel below the canvas. On the left side of the script library is a list of all the experiences your account has access to, meaning you can access scripts across multiple expereinces. There is also a **Public** category near the top of this list.  It gives you access to of all of the currently available public scripts.
```
Comment (liam-philiben) [inline docs/Scripting/PublicScripts.md:11]: remove the first "of" in the last sentence (so "it gives you access to all of the currently available public scripts")

File: docs/Scripting/PublicScripts.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Public Scripts
+
+To use a public script, simply select the element you want to attach it to, then drag the script to the **Scripts** section at the bottom of the inspector.  Once you have attatched your script, you can click on the **Variables** tab to see any exposed variables the script may have.
+
+![Drag public orbit script onto planet to make it orbit.](/img/product/editor/public_scripts/PublicScripts_OrbitScript.gif)
+
+### Inspector Variables
+
+Using the **Variables** tab in the inspector is a quick and easy way to change script parameters without needed to open the script or do any coding.  Not every script will have variables exposed to the inspector in this way.  Particularly for custom scripts you write yourself, you would have to intentionally include syntax to expose variables to the inspector.  But most public scripts will have variables that can be modified from the inspector.
```
Comment (liam-philiben) [inline docs/Scripting/PublicScripts.md:22]: should be "without needing to" instead of "needed to"

File: docs/Scripting/PublicScripts.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- **Spin Speed around Z Axis** - this is a field you can type a number into which influences how fast the element will spin around its Z axis.
+
+#### Scale_Basic
+
+- **Test Scale** - this is a checkbox which can trigger the scale change.
+- **X Scaling** - this is a field you can type a number into which influences scaling along the X axis.
+- **Y Scaling** - this is a field you can type a number into which influences scaling along the Y axis.
+- **Z Scaling** - this is a field you can type a number into which influences scaling along the Z axis.
+- **Scale Duration in seconds** - this is a field you can type into which determines how long the scaling will take to complete.
+- **Scale Trigger Message** - this is a text field you can type a message for this script to listen for to trigger the scaling to start.
```
Comment (liam-philiben) [inline docs/Scripting/PublicScripts.md:48]: a bit confusing to read (seems like a lot of "fors" haha), maybe "this is a text field where you can type a message for this script to listen to in order to trigger scaling"?

**Review by liam-philiben — APPROVED** [review]
mainly just typos/grammatical errors/sentence structure stuff (and very little of them at that), but otherwise looks good!

**Review by jsundra — APPROVED** [review]
👍

**Pull request merged**